src/main/groovy/com/twcable/grabbit/jcr/JcrPropertyDecorator.groovy

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2015 Time Warner Cable, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twcable.grabbit.jcr
+
+import com.google.protobuf.ByteString
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+
+import javax.annotation.Nonnull
+import javax.jcr.Property
+import javax.jcr.Property as JCRProperty
+import javax.jcr.Value
+
+import static javax.jcr.PropertyType.BINARY
+import static org.apache.jackrabbit.JcrConstants.JCR_LASTMODIFIED
+import static org.apache.jackrabbit.JcrConstants.JCR_MIXINTYPES
+import static org.apache.jackrabbit.JcrConstants.JCR_PRIMARYTYPE
+import com.twcable.grabbit.proto.NodeProtos.Property as ProtoProperty
+import com.twcable.grabbit.proto.NodeProtos.Property.Builder as ProtoPropertyBuilder
+import com.twcable.grabbit.proto.NodeProtos.Value as ProtoValue
+import com.twcable.grabbit.proto.NodeProtos.Value.Builder as ProtoValueBuilder
+
+
+@CompileStatic
+@Slf4j
+class JcrPropertyDecorator {
```

Comment:
the `JCRNodeDecorator` has `JCR` .. this has `Jcr` .. eitherway is fine I think (though I think `Jcr` is what we have elsewhere .. so may be rename the `JCRNodeDecorator` to `JcrNodeDecorator` ...?